libs/remix-analyzer/package.json

```diff
@@ -21,14 +21,14 @@
     }
   ],
   "dependencies": {
-    "@ethereumjs/block": "^10.0.0-rc.1",
-    "@ethereumjs/tx": "^10.0.0-rc.1",
-    "@ethereumjs/util": "^10.0.0-rc.1",
-    "@ethereumjs/vm": "^10.0.0-rc.1",
+    "@ethereumjs/block": "^10.0.0",
+    "@ethereumjs/tx": "github:yann300/ethereumjs-vm#fixV10",
```

Comment:
do you want to keep this dep to your repo?

Reply:
Yes, this is a fix that will be soon released by them.. but it's not yet..